messages/messages.md

```diff
@@ -93,3 +93,11 @@ Set varargs with this format: key=value or key="value with spaces".
 # error.DuplicateArgument
 
 Found duplicate argument %s.
+
+# warning.arrayInputFormat
+
+The input format for array arguments has changed. Use this format: --arrayFlag value1 --arrayFlag value2 --arrayFlag value3
+
+# arrayStyleFlagsDeprecated
+
+The array style flags are deprecated. Use oclif/core Flags.string({multiple: true}) instead.
```

Comment:
This is emitted once for EACH array flag on the command.

I don't think the wording is quite right. Without getting too long it should serve as warning for the dev, if the line through the name isn't enough) and as nudge for the end user.